README.md

```diff
@@ -489,12 +489,12 @@ under which circumstances:
 | HEAD |  |   |  Read |
 | GET |  |   |  Read |
 | POST | /foo/ | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ gets created | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
-| PATCH | /foo/bar | patch instructions contain deletions | Write on /foo/bar |
+| PUT | /foo/bar | /foo/ exists, will contain /foo/bar | Append or Write on /foo/ + Write on resulting resource |
+| PUT | /foo/bar | /foo/ gets created, will contain /foo/bar | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
+| PATCH | /foo/bar | patch instructions contain deletions | Read and Write on /foo/bar |
 | PATCH | /foo/bar | patch instructions contain only insertions | Append or Write on /foo/bar |
```

Comment:
For `PATCH` instructions that only contain insertions, I can imagine that `Read` would be useful or necessary in an implementation. For example, given the document:

```turtle
<> a <https://example.com/MyType> .
```
the SPARQL Query:

```sparql
INSERT {
    <> a <https://example.com/MyType> .
} WHERE {}
```

would result in no change to the original Graph, but that cannot be determined unless the document is first read into a Graph.